### DISCUSSION TITLE
Kiosks should have a universal "forward" direction

### DISCUSSION THREAD

**Discussion by Asnro** [tsunamayo/Starship-EVO#4599] [Suggestions]
The station I've been building has kiosks situated along a narrow-ish hall, and due to the non-standard directions half the kiosks face the walls and half face the walkway.

![kuva](https://user-images.githubusercontent.com/14952174/155292356-a9847b1d-7d27-401e-b818-706ea24f99cc.png)

It would be helpful for builders if kiosks had a standard direction "facing the street", so to say.

**Comment by tsunamayo** [comment]
@Asnro Hi, I think that if you turn off Randomize rotation in the docking port they wont rotate. Of course you have to be careful on how you built the docking port, and if the kiosks builder didnt used the same forward convention it wont work.
Thanks

**Reply by Asnro** [reply]
Hi, Randomize rotation is turned off and I experimented with several docking port orientations. The issue is with kiosk builders not using the same forward direction.
